src/Integration/Persistence/ISolutionBindingDataReader.cs

```diff
@@ -33,12 +34,19 @@ internal interface ISolutionBindingDataReader
         BoundSonarQubeProject Read(string configFilePath);
     }
 
+    [Export(typeof(ISolutionBindingDataReader))]
     internal class SolutionBindingDataReader : ISolutionBindingDataReader
     {
         private readonly ISolutionBindingFileLoader solutionBindingFileLoader;
         private readonly ISolutionBindingCredentialsLoader credentialsLoader;
 
-        public SolutionBindingDataReader(ISolutionBindingFileLoader solutionBindingFileLoader, ISolutionBindingCredentialsLoader credentialsLoader)
+        [ImportingConstructor]
+        public SolutionBindingDataReader(ICredentialStoreService credentialStoreService, ILogger logger)
```

Comment:
Both the `ICredentialStoreService` and `ILogger` are already MEF exports, so making the data reader exportable is straightforward.